src/Symfony/Component/Translation/Bridge/Loco/LocoProvider.php

```diff
@@ -91,92 +84,137 @@ public function read(array $domains, array $locales): TranslatorBag
     {
         $domains = $domains ?: ['*'];
         $translatorBag = new TranslatorBag();
+        $responses = [];
 
         foreach ($locales as $locale) {
             foreach ($domains as $domain) {
-                $response = $this->client->request('GET', sprintf('export/locale/%s.xlf?filter=%s&status=translated', $locale, $domain));
+                $responses[] = [
+                    'response' => $this->client->request('GET', sprintf('export/locale/%s.xlf?filter=%s&status=translated', $locale, $domain)),
+                    'locale' => $locale,
+                    'domain' => $domain,
+                ];
+            }
+        }
 
-                if (404 === $response->getStatusCode()) {
-                    $this->logger->error(sprintf('Locale "%s" for domain "%s" does not exist in Loco.', $locale, $domain));
-                    continue;
-                }
+        foreach ($responses as $response) {
+            $locale = $response['locale'];
+            $domain = $response['domain'];
+            $response = $response['response'];
 
-                $responseContent = $response->getContent(false);
+            if (404 === $response->getStatusCode()) {
+                $this->logger->error(sprintf('Locale "%s" for domain "%s" does not exist in Loco.', $locale, $domain));
```

Comment:
is it actually something to be logged at the `error` level or could it be a lower level ?

Reply:
I took it down to the `warning` level, as it could be an typo in the command option `--locale`.